src/buttonDelivery.styles.js

```diff
@@ -0,0 +1,8 @@
+export const buttonDetailStyle = {
+  color: "#000",
+  textTransform: "none",
+  boxShadow: `0px 1px 4px  rgba(0, 0, 0, 0.15)`,
+  minWidth: 122,
+  minHeight: 40,
+
```

Comment:
Usually you won't need to define the width and the height for a button, you should only care about the padding and the font size and the width and the height will be calculated automatically